gui/executionGraphGui/client/apps/eg/src/app/components/file-browser/file-browser.component.ts

```diff
@@ -92,14 +105,37 @@ export class FileBrowserComponent implements OnInit {
   }
 
   public openFile(file: FileInfo) {
+    assertThat(this.mode === FileBrowserMode.Open);
```

Comment:
Where is the assertEquals method? ;)

Reply:
Also, where is the assertion error message?

Reply:
not needed, assertThat is a wrapper in the GUI to be switched of if desired( had no better naming)...
ok message should not be optional :+1: